lib/sandboxed_renderer/init.js

```diff
@@ -113,6 +113,12 @@ function preloadRequire (module) {
 // Process command line arguments.
 const { hasSwitch } = process.electronBinding('command_line');
 
+// Similar to nodes --expose-internals flag, this exposes electronBinding so
+// that tests can call it to get access to some test only bindings
+if (hasSwitch('unsafely-expose-electron-internals-for-testing')) {
+  preloadProcess.electronBinding = process.electronBinding;
+}
```

Comment:
since the test in question relies on `DCHECK_IS_ON` (i.e. won't run on release builds), do we want to make this flag also only active when `DCHECK_IS_ON`?

Reply:
@nornagon That `DCHECK_IS_ON` flag isn't available in JS land

Reply:
@MarshallOfSound not with that attitude ;)